galaxy_ng/app/api/v3/urls.py

```diff
@@ -31,6 +31,17 @@
 ]
 
 urlpatterns = [
+    path("", viewsets.RepoMetadataViewSet.as_view({"get": "retrieve"}), name="repo-metadata"),
+    path(
+        'collections/all/',
+        viewsets.UnpaginatedCollectionViewSet.as_view({'get': 'list'}),
+        name='all-collections-list'
+    ),
+    path(
```

Comment:
Design question, could the unpaginated collection list viewsets be implemented with a query param to CollectionViewSet and CollectionVersionViewSet?

ie, in lieu of new viewsets.

I think either is fine, but was curious if there was a reason for one or the other.

Reply:
@alikins we just followed what `pulp_ansible` implemented to keep our side conforming with that. I suggested having a dynamic endpoint e,g: `collections/?per_page=all&exclude=manifest&exclude=files` but the way it is was already implemented.

I also think that a dynamic endpoint would be hard to implement and test.

@fao89 any input?

Reply:
no special reason, when I was implementing I didn't think about a query_param to change the pagination, but as @rochacbruno mentioned it could be hard to test, `CollectionViewSet` is getting bigger and bigger